requirements.txt

```diff
@@ -1,8 +1,11 @@
 # Core
-Django==1.10.3
+Django==1.10.6
```

Comment:
how do you keep this so nicely formatted... `pip freeze` just outputs an alphabeticallysorted list with all comments and whitespace removed O_o